sa/type-converter.go

```diff
@@ -38,6 +38,16 @@ func (tc BoulderTypeConverter) ToDb(val interface{}) (interface{}, error) {
 		return string(t), nil
 	case core.OCSPStatus:
 		return string(t), nil
+	case *core.AcmeURL:
```

Comment:
These shouldn't be required. core.AcmeURL is already a string and a jose.JWS shouldn't ever get in here since that data should be a []byte of marshaled json.